grid/recruitment/candidates/profile.supr

```diff
@@ -139,6 +140,108 @@ usecase CreateCandidate {
   }
 }
 
+"""
+Get Stage Changes
+Retrieves a list of stages for specific candidate throughout his recruitment process. The candidate does not have to be applied for a job.
+"""
+usecase GetStageChanges {
```

Comment:
suggestion: Move GetStageChanges into separate profile `recruitment/get-stage-changes`. There reasons are the a same as in case `CreateLead` use case. Having single use case per profile is the best approach for ATS domain harmonisation. Look at the ATS domain providers comparison table and supported use cases across all of them.